CHANGELOG/schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "message": {
+      "type": "string",
+      "description": "Message of the changelog",
+      "minLength": 1,
+      "maxLength": 1000
+    },
+    "type": {
+      "type": "string",
+      "description": "Changelog type",
+      "enum": [
+        "feature",
+        "bugfix",
+        "dependency",
+        "deprecation",
+        "breaking_change"
+      ]
+    },
+    "scope": {
+      "type": "string",
+      "description": "Changelog scope",
+      "enum": [
+        "Core",
+        "Plugin",
+        "PDK",
+        "Admin API",
+        "Performance",
+        "Configuration",
+        "Clustering"
+      ]
+    },
+    "pr": {
+      "type": ["integer", "string", "null"],
+      "description": "Associated GitHub PR",
+      "pattern": "^(\\d+|\\w+/\\w+:\\d+)$",
+      "examples": ["1", "torvalds/linux:1"]
+    },
```

Comment:
What do you think about making `pr` an array so that we can link multiple PRs to a changelog entry? It could be handled the same as `issues` or `jiras`.

Reply:
Added another field `prs`. But I haven't decided whether to remove the single one `pr` yet.

Reply:
I don't have a strong preference, but for simplicity's sake I think I'd vote for removing one of them and just having one field (whether it's `pr` or `prs`).

Relevant idea: you could also make the tool accept either a single item or an array for pr/issue/jira references, so that this:

```yaml
message: my changelog entry
type: feature
scope: core
prs:
  - 123
  - 456
issues: 789
jiras: abc-123
```

...is valid and coerces to this:

```yaml
message: my changelog entry
type: feature
scope: core
prs:
  - 123
  - 456
issues:
  - 789
jiras:
  - abc-123
```

Up to you though--I think things are in good shape here as-is.

Reply:
I removed `pr` to keep things simple first.